[DM-3747] Announcement snmp no longer supported (#2223)
SoftwareAG/c8y-docs: content/change-logs/device-management/SNMP-removed-from-docs.md

---

> date: 2024-06-26T00:00:00.000Z

Reviewer: ```suggestion
date: 2024-07-08T00:00:00.000Z
```

> title: Deprecation of SNMP

Reviewer: ```suggestion
title: Moving SNMP documentation
```

> product_area: Device management

Reviewer: ```suggestion
product_area: Device Management
```

change_type:

>   - value: change-inv-3bw8e

Reviewer: ```suggestion
  - value: change-2c7RdTdXo4
```

>     label: Announcement

Reviewer: ```suggestion
    label: Improvement
```

component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
---

> SNMP functionality is considered deprecated (since which version?) and the respective documentation have been moved to GitHub repository: [cumulocity-examples](https://github.com/SoftwareAG/cumulocity-examples/snmp). This means that the feature is no longer actively supported by the product team.

Reviewer: ```suggestion
The documentation of the SNMP device protocol has been moved from the [user documentation](https://cumulocity.com/docs) to the public GitHub repository: [cumulocity-examples](https://github.com/SoftwareAG/cumulocity-examples/snmp). 
```
